build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
 
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = System.getProperty("opensearch.version", "1.3.1-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "1.3.2-SNAPSHOT")
```

Comment:
Do we need to wait for infra team to open the version bumping ticket first, then increment the version?

Reply:
Infra team has bumped version in `1.3` branch https://github.com/opensearch-project/ml-commons/pull/270